src/prpy/serialization.py

```diff
@@ -300,12 +330,16 @@ def deserialize_environment(data, env=None, purge=False, reuse_bodies=None):
             deserialization_logger.debug('Purging body "%s".', body.GetName())
             env.Remove(body)
 
+    # Create a or_ordf module on demand
+    urdf_module_getter = UnitaryMemoizer(lambda: openravepy.RaveCreateModule(env,'urdf'))
```

Comment:
Throw an exception if creating the `urdf` module fails, i.e. `RaveCreateModule` returns `None`.